File: desktop/Models/Question.cs
namespace StadiumCompany.Models;

public enum AnswerType
{
    TRUE_FALSE,
    MULTIPLE_CHOICE
}

public class Question
{
    public int Id { get; set; }
    public int QuestionnaireId { get; set; }
    public int Number { get; set; }
    public string Label { get; set; } = string.Empty;
    public AnswerType AnswerType { get; set; }

    // Navigation properties
    public Questionnaire? Questionnaire { get; set; }
    public List<Answer> Answers { get; set; } = [];
}

File: web/app/Models/Question.php
<?php

namespace App\Models;

use App\Enums\AnswerType;
use Illuminate\Database\Eloquent\Model;

class Question extends Model
{
    protected $table = 'questions';

    public $timestamps = false;

    protected $fillable = [
        'questionnaire_id',
        'number',
        'label',
        'answer_type',
    ];

    protected function casts(): array
    {
        return [
            'answer_type' => AnswerType::class,
        ];
    }

    public function questionnaire()
    {
        return $this->belongsTo(Questionnaire::class);
    }

    public function answers()
    {
        return $this->hasMany(Answer::class)->orderBy('id');
    }
}
